tritium-metrics/src/main/java/com/palantir/tritium/event/metrics/TaggedMetricsServiceInvocationEventHandler.java

```diff
@@ -67,7 +67,10 @@ private static BooleanSupplier getEnabledSupplier(final String serviceName) {
     }
 
     @Override
-    public final InvocationContext preInvocation(Object instance, Method method, Object[] args) {
+    public final InvocationContext preInvocation(
```

Comment:
Should we annotate the return of `preInvocation` as `@Nullable`?

Reply:
So the `com.palantir.tritium.event.InvocationEventHandler#preInvocation` API actually requires this to return a non-null, though implementations can potentially throw, in which case we protect against the throwing case, and skip any handlers whose `preInvocation` and treat them as disabled. A followup PR will make this more explicit and take advantage of that fact.